app/src/main/java/com/keylesspalace/tusky/adapter/ComposeAutoCompleteAdapter.java

```diff
@@ -159,6 +163,31 @@ public View getView(int position, @Nullable View convertView, @NonNull ViewGroup
                     ((TextView) view).setText(formatHashtag(result));
                 }
                 break;
+
+            case EMOJI_VIEW_TYPE:
+                EmojiViewHolder emojiViewHolder;
+                if (convertView == null) {
+                    //noinspection ConstantConditions
+                    view = ((LayoutInflater) context
+                            .getSystemService(Context.LAYOUT_INFLATER_SERVICE))
+                            .inflate(R.layout.item_autocomplete_emoji, parent, false);
+                }
+                if (view.getTag() == null) {
+                    view.setTag(new EmojiViewHolder(view));
+                }
+                emojiViewHolder = (EmojiViewHolder) view.getTag();
+
+                EmojiResult emojiResult = ((EmojiResult) getItem(position));
+                if (emojiResult != null) {
+                    Emoji emoji = emojiResult.emoji;
+                    String format = context.getString(R.string.emoji_shortcode_format);
+                    String formattedShortcode = String.format(format, emoji.getShortcode());
```

Comment:
`context.getString` takes format arguments, you could save a line of code if you want